src/views/photos_week_loop/index.php

```diff
@@ -10,31 +10,51 @@
 <div class="archive-query wp-block-template-part">
 	<div class="wp-block-query ucsc__post-query-loop has-global-padding is-layout-constrained wp-block-query-is-layout-constrained">
 		<?php if ( $posts->have_posts() ) : ?>
-			<ul>
-			<?php while ( $posts->have_posts() ) :
-				$posts->the_post();  ?>
-				<li class="wp-block-post post-<?php echo get_the_ID(); ?> photo_of_the_week type-photo_of_the_week status-publish hentry">
-					<div>
-						<?php echo $c->get_image() ?>
-					</div>
-					<div>
-						<h2><?php echo get_the_title() ?></h2>
+			<?php while ( $posts->have_posts() ) : ?>
+				<?php $posts->the_post(); ?>
+			<figure class="wp-block-post post-<?php echo get_the_ID(); ?> photo-of-the-week type-photo_of_the_week status-publish hentry">
+				<?php echo $c->get_image(); ?>
+				<figcaption>
+					<div class="photo-of-the-week__caption">
+						<h2 class="photo-of-the-week__title has-ucsc-primary-blue-color has-two-font-size">
+							<?php echo get_the_title(); ?>
+						</h2>
 						<?php $author = get_field( Photo_Of_The_Week_Meta::PHOTOGRAPHER, get_the_ID() ); ?>
 						<?php if ( ! empty( $author ) ) : ?>
+						<p class="photo-of-the-week__author">
+							<svg width="16" height="16" fill="none" stroke="currentColor" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round" xmlns="http://www.w3.org/2000/svg">
+								<path d="M2.5.5h4M9.5 12.5a3 3 0 1 0 0-6 3 3 0 0 0 0 6Z"/>
+								<path d="M14.5 15.5h-13a1 1 0 0 1-1-1v-10a1 1 0 0 1 1-1h13a1 1 0 0 1 1 1v10a1 1 0 0 1-1 1Z"/>
+							</svg>
 							<?php echo $author; ?>
+						</p>
 						<?php endif; ?>
-						<a href="<?php echo get_the_permalink( get_the_ID() ); ?>" target="_blank">
-							<?php echo esc_html__( 'Download Image', 'ucsc' ); ?>
-						</a>
 					</div>
-				</li>
+					<a class="photo-of-the-week__download-link" href="<?php echo get_the_permalink( get_the_ID() ); ?>" target="_blank">
+						<svg width="16" height="16" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" xmlns="http://www.w3.org/2000/svg">
+							<path d="M8 .5v11M3.5 7 8 11.5 12.5 7M.5 15.5h15" />
+						</svg>
+						<?php echo esc_html__( 'Download Image', 'ucsc' ); ?>
+					</a>
+				</figcaption>
+			</figure>
 			<?php endwhile; ?>
-			</ul>
+	
 			<!-- Pagination -->
 			<nav class="wp-block-query-pagination is-content-justification-center is-layout-flex wp-container-core-query-pagination-is-layout-1 wp-block-query-pagination-is-layout-flex">
-				<?php
-				echo $c->get_pagination( $posts, $paged );
-				?>
+				<?php if ( $paged === 1 ) : ?>
```

Comment:
I believe we should not show buttons if we don't have other pages. So you may want to adjust your statement to the 
`<?php if ( $paged === 1 && $posts->max_num_pages > 1 ) : ?>` 
Same will applies to next button
`<?php if ( $paged === $posts->max_num_pages && $posts->max_num_pages > 1 ) : ?>`

Reply:
The design calls for the previous and next buttons to always be present in the pager, but disabled when they don't do anything. As I understand it, the `get_pagination()` logic leaves off the previous button on the first page and leaves off the next button on the last page, so those are the cases I tried to cover here.

Reply:
Page looks a bit odd when we have 2 disabled buttons and only 1 page. get_pagination will hide any buttons if there is only 1 page
Leave it as is but I'd clarify this with design and strategy

Reply:
Agree that it looks odd, but it is what's shown in the design so I was gonna let  design bring it up in DQA. I'll call it out specifically in the testing instructions, Thank you!